src/reconcile/rec_visibility.c

```diff
@@ -471,8 +471,13 @@ __rec_validate_upd_chain(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_UPDATE *s
             prev_upd->start_ts == prev_upd->durable_ts || !WT_TIME_WINDOW_HAS_STOP(&vpack->tw) ||
             prev_upd->durable_ts >= vpack->tw.durable_stop_ts,
           "Durable timestamps cannot be out of order for prepared updates");
```

Comment:
Me and tod discussed about changing these assert as

if (WT_TIME_WINDOW_HAS_STOP(&vpack->tw))
...
else
...

This improves readability

Reply:
Not done.
1. When we discussed this I misread this. The first WT_ASSERT_ALWAYS() is checking vpack->tw.durable_start_ts. The second WT_ASSERT_ALWAYS() is checking vpack->tw.durable_stop_ts.

2. If WT_ASSERT_ALWAYS() fails the assert condition is printed. Moving part of the condition out of the WT_ASSERT_ALWAYS() prints just part of the assert condition if the assert fails.

I did change the asserts to have different comments.

Reply:
The part that is moved out of the assert is not important.